custom_components/wyzeapi/switch.py

```diff
@@ -187,7 +191,10 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     @property
     def name(self):
         """Return the display name of this switch."""
-        return self._device.nickname
+        if type(self._device) is Camera:
+            return f"{self._device.nickname} Power"
```

Comment:
Changing the name for clarity since we have 2 switches per camera now.

Reply:
nvm I see now